.github/workflows/staging-build-data-prepper.yml

```diff
@@ -0,0 +1,71 @@
+name: Process DATAPREPPER Artifacts
+
+on:
+#  schedule:
+#    - cron: '0 10 * * *'
+  repository_dispatch:
+    types: [staging-build-data-prepper]
+
+jobs:
+  sync-dataprepper-artifacts:
+    name: sync-dataprepper-artifacts
+    runs-on: [ubuntu-18.04]
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: us-east-1
+
+    - name: S3 sync artifacts
+      run: |
+        #!/bin/bash
+        S3_BUCKET_PROD="artifacts.opendistroforelasticsearch.amazon.com"
+        S3_BUCKET_DATAPREPPER="artifacts.dataprepper.amazon.com"
+        TARGET_DIR=`./bin/plugins-info data-prepper zip --require-install-false`
+        TAG_VERSION="v$(./bin/plugins-info data-prepper cutversion --require-install-false | tr ' ' '\n' | uniq | sort -V | head -n 1)"
+
+        echo $TAG_VERSION
+        aws s3 sync s3://$S3_BUCKET_DATAPREPPER/ s3://$S3_BUCKET_PROD/$TARGET_DIR/ --exclude "*" --include "data-prepper-jdk-${TAG_VERSION}*" --quiet
+        aws s3 sync s3://$S3_BUCKET_DATAPREPPER/ s3://$S3_BUCKET_PROD/$TARGET_DIR/ --exclude "*" --include "data-prepper-${TAG_VERSION}*" --quiet
+        aws s3 ls s3://$S3_BUCKET_PROD/$TARGET_DIR/ | grep $TAG_VERSION
+        aws cloudfront create-invalidation --distribution-id E1VG5HMIWI4SA2 --paths "/$TARGET_DIR/*"
+
+  sync-dataprepper-docker-images:
+    name: sync-dataprepper-docker-images
+    runs-on: [ubuntu-18.04]
+    strategy:
+      fail-fast: false
```

Comment:
Not required

Reply:
Not required to delete it either 👍

Reply:
Why to keep extra code around when not required? It can be confusing sometimes

Reply:
Because at some times it will need to consume JDK 15 as well why remove it now and forget later.